kahuna/public/js/components/gu-lazy-table/gu-lazy-table.js

```diff
@@ -97,6 +97,13 @@ lazyTable.controller('GuLazyTableCtrl', ['range', function(range) {
             filter(({$start, $end}) => $start !== -1 && $end !== -1).
             // Ignore if $start after $end (incomplete combine$ state)
             filter(({$start, $end}) => $start <= $end).
+            // Max query size of 200
+            map(({$start, $end})=> {
+              if ($end - $start > 200) {
```

Comment:
🤔 Should this be `$end - $start <= 200`?

Reply:
oops, it was right the first time. 

the equal is equivalent though, so I'm not particular